test/000_fullchain.test.ts

```diff
@@ -174,44 +173,380 @@ describe('Integration tests', function () {
             //   - Worker:
```

Comment:
What do you think of removing this comment block ? Balances check seems clearer now.

Reply:
[remove comment block](https://github.com/iExecBlockchainComputing/PoCo/pull/163/commits/7aa37cbcf474ea39b7ff2b33bdee8064736fef5c)